cli/cmd/ps/ps_services.go

```diff
@@ -121,13 +121,17 @@ func (p *Ps) services(ctx *clicontext.CLIContext) error {
 
 		weight := 0
 		if service.(*riov1.Service).Status.ComputedWeight != nil && *service.(*riov1.Service).Status.ComputedWeight > 0 {
-			totalWeight := 0
-			for _, subService := range services {
-				if service.(*riov1.Service).Spec.App == subService.(*riov1.Service).Spec.App && subService.(*riov1.Service).Status.ComputedWeight != nil {
-					totalWeight += *subService.(*riov1.Service).Status.ComputedWeight
+			if service.(*riov1.Service).Spec.App == "" {
```

Comment:
@cbron You should always use `appAndVersion` method to get app.

Reply:
Fixed